cc/trees/tree_synchronizer.cc

```diff
@@ -219,6 +219,11 @@ void TreeSynchronizer::PushLayerProperties(
       commit_state.layers_that_should_push_properties.end();
   for (auto it = source_layers_begin; it != source_layers_end; ++it) {
     auto* source_layer = *it;
+    // https://linear.app/replay/issue/RUN-1229
+    recordreplay::Assert(
```

Comment:
hi @MichaReiser,
This is awesome stuff!

When I last looked, I did not pay enough attention, and forgot to mention: When adding new or revisiting old `Assert`s, we want to start adding a more specific label. We use these labels to do some automagical crash filtering in http://admin/crash/filters

Currently the label format is `[RUN-X]`, where `X` refers to the original crash report issue.

In the future, we want to move toward `[RUN-X-Y]`, where `Y` is the child issue that the PR is filed under; hopefully by next week. It is currently blocked by https://linear.app/replay/issue/SCS-784/[crash-buckets]-allow-more-flexible-labels